SchedulePlatform.Api/Mappings/CustomerMappingExtension.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Microsoft.IdentityModel.Tokens;
+using SchedulePlatform.Api.Models;
+using SchedulePlatform.Models.Entities;
+
+namespace SchedulePlatform.Api.Mappings
+{
+	public static class CustomerMappingExtensions
+	{
+		public static Customer Map(this Customer customer, CustomerPatchModel model)
+		{
+            if (!string.IsNullOrEmpty(model.FirstName))
+            {
+                customer.FirstName = model.FirstName;
+            }
+
```

Comment:
Consistency is a key element when coding. I see that between these ifs you do have a new line but on the ones in the file above you don't. This basically breaks consistency and should make you think about which one of the two approaches you want to use -> this way you will realize that one approach is probably better than the other and you will start using it everywhere.